frappedesk/api/ticket.py

```diff
@@ -139,7 +142,7 @@ def assign_ticket_type(ticket_id, type):
 
 		return ticket_doc
 
-@frappe.whitelist(allow_guest=True)
+@frappe.whitelist()
 def assign_ticket_status(ticket_id, status):
```

Comment:
There will be an issue here
if the ticket is created via email and then the user goes to the customer portal and tries to close the ticket, it will show an error because, writing permission for a ticket is only given to the creator of the ticket, but since the ticket was created via email the creator is set as admin.